jax/interpreters/xla.py

```diff
@@ -142,7 +142,12 @@ def xla_primitive_callable(prim, *abstract_args, **params):
 def primitive_computation(prim, *xla_shapes, **params):
   backend = params.get('backend', None)
   new_params = {k: params[k] for k in params if k != 'backend'}
-  c = xb.make_computation_builder("primitive_computation")  # TODO(name)
+  c = xb.make_computation_builder("primitive_computation_{}".format(prim.name))
+  c.SetOpMetadata(xc.CurrentSourceInfoMetadata(
+      op_type=prim.name,
+      op_name=prim.name,
+      skip_frames=6
```

Comment:
`primitive_computation` is cached, so this frame information might not be right for cached calls (if they're called from somewhere else). I'm also worried that extracting the stack info will slow things down -- the caching might make it acceptable, but this was a huge performance bottleneck in TF graph mode. Maybe just create an OpMetadata with the op_type and op_name and leave the file info blank for now?